coreneuron/gpu/nrn_acc_manager.cpp

```diff
@@ -984,6 +1157,27 @@ void nrn_sparseobj_copyto_device(SparseObj* so) {
 #endif
 }
 
+void nrn_sparseobj_delete_from_device(SparseObj* so) {
+#ifdef _OPENACC
+    if (nrn_threads[0].compute_gpu == 0) {
+        return;
+    }
```

Comment:
Just a note: `SparseObj* so` argument to this function is for every thread object in `nrn_threads` array. It is technically possible to have some NrnThread running on CPUs and some on GPUs (currently we don't have CLI option though!). In this case, checking just 0th thread `nrn_threads[0].compute_gpu` won't be sufficient. We can add a comment here (or in follow-up PR)

Reply:
Done in 5456f65.